airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -362,15 +362,13 @@ class GoogleDisplayVideo360RunReportOperator(BaseOperator):
 
     template_fields = (
         "report_id",
-        "params",
         "impersonation_chain",
     )
 
     def __init__(
         self,
         *,
         report_id: str,
-        params: Dict[str, Any],
```

Comment:
Same here -- it's a name clash  but has a different meaning/use to the base params.

Maybe both of these should be stored as `self._params`?

Reply:
Not sure if I understood this. As this class defines and uses `params` there is no way to access base params so there is always a single accessible params only.